experimental/traffic-portal/src/app/api/cdn.service.spec.ts

```diff
@@ -129,6 +129,21 @@ describe("CDNService", () => {
 		await expectAsync(responseP).toBeResolved();
 	});
 
+	it("queues a CDN by ID", async () => {
+		const responseP = service.queueCDNUpdates(cdn, "queue");
```

Comment:
This doesn't queue updates using an ID, it uses a whole CDN object. Because the `number` case isn't covered, we still aren't hitting 100% of statements - but even if you change that we won't hit 100% of branches because then the passing-a-whole-CDN-object case won't be under test.

Reply:
> This doesn't queue updates using an ID, it uses a whole CDN object. Because the `number` case isn't covered, we still aren't hitting 100% of statements - but even if you change that we won't hit 100% of branches because then the passing-a-whole-CDN-object case won't be under test.

Testing both ID and CDN object in 33499dd69f